StatPlugin.cs

```diff
@@ -1,50 +1,79 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using ServiceStack.Redis;
 
 namespace DNWS
 {
-  class StatPlugin : IPlugin
-  {
-    protected static Dictionary<String, int> statDictionary = null;
-    public StatPlugin()
+    class StatPlugin : IPlugin
     {
-      if (statDictionary == null)
-      {
-        statDictionary = new Dictionary<String, int>();
+        protected static Dictionary<String, int> statDictionary = null;
+        protected static RedisManagerPool redisManager = null;
+        public StatPlugin()
+        {
+            string redisHost = Environment.GetEnvironmentVariable("redisHost");
```

Comment:
why don't you just call string redisHost = "redis" and let Docker's dns do the name resolution? what's wrong with that?